Examples/LambdaFunctions/Sources/LambdaAPI/ProductService.swift

```diff
@@ -0,0 +1,104 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftAWSLambdaRuntime open source project
+//
+// Copyright (c) 2020 Apple Inc. and the SwiftAWSLambdaRuntime project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftAWSLambdaRuntime project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import AWSDynamoDB
+import NIO
+
+public enum APIError: Error {
+    case invalidItem
+    case tableNameNotFound
+    case invalidRequest
+    case handlerNotFound
+}
+
+extension Date {
```

Comment:
reuse from Events module?

Reply:
Used after refactor of Operation
ff3326e